doc/source/hla.rst

```diff
@@ -0,0 +1,21 @@
+.. hla:
+
+HLA alignment API
```

Comment:
The Mission Office has expressed an interest to NOT refer to these new products as HLA, since that implies a product generated by the separate HLA effort.  Instead, we have been asked to refer to this code and the products as 'Enhanced Pipeline Products' in the documentation, at least until some better term is decided upon by the Mission Offices.